skyportal/models.py

```diff
@@ -463,6 +463,7 @@ class Photometry(Base):
     altdata = sa.Column(JSONB)
     upload_id = sa.Column(sa.String, nullable=False,
                           default=lambda: str(uuid.uuid4()))
+    alert_id = sa.Column(sa.BigInteger, nullable=True, unique=True)
```

Comment:
Is it ok to make it unique and allow to be None? (I think yes, but just checking)

Reply:
@dmitryduev yes, null isn't counted as a value as far as the unique constraint is concerned